### PR TITLE
ETO users

### PR DESCRIPTION
**Pull request by krzkaczor**
routing for both investors and issuers, tweak interfaces to contain user type property

### CODE REVIEW

File: app/modules/auth/reducer.spec.ts [hunk 1]
```diff
@@ -7,7 +7,9 @@ describe("auth > selectors", () => {
     it("should return true for authorized users", () => {
```
Comment (Moejoe90) [inline app/modules/auth/reducer.spec.ts]: Shouldn't the test somehow reflect that we are only dealing with investors no users anymore?

File: app/modules/auth/sagas.ts [hunk 2]
```diff
@@ -39,10 +39,12 @@ export async function loadUserPromise({
       newEmail: walletMetadata.email,
       salt: walletMetadata.salt,
       backupCodesVerified: false,
+      type: "investor",
```
Comment (Moejoe90) [inline app/modules/auth/sagas.ts]: Are you planning to register `issuers` in the same saga? or create a complete different saga for them

File: app/lib/api/users/UsersApi.ts [hunk 3]
```diff
@@ -42,11 +42,13 @@ export class UsersApi {
...
     }
 
+    response.body.type = "investor"; // @todo remove it when api is done
+
     return response.body;
   }
 
   public async verifyUserEmail(userCode: IVerifyEmailUser): Promise<IUser> {
-    const response = await this.httpClient.put<IVerifyEmailUser>({
+    const response = await this.httpClient.put<IUser>({
```
Comment (Moejoe90) [inline app/lib/api/users/UsersApi.ts]: :+1:

File: app/modules/auth/reducer.spec.ts (hunk 1 above)
Comment (krzkaczor) [reply app/modules/auth/reducer.spec.ts]: this selector doesnt use investor type at all sooo i guess its fine

File: app/modules/auth/sagas.ts (hunk 2 above)
Comment (krzkaczor) [reply app/modules/auth/sagas.ts]: probably the same saga — just paramettrized. i am gonna take a closer look when working on login

File: app/components/AppRouter.tsx [hunk 4]
```diff
@@ -1,11 +1,17 @@
...
+
   root: "/",
+
   register: "/register",
+  etoRegister: "/register-eto",
   login: "/login",
-  kyc: "/kyc",
+  etoLogin: "/login-eto",
   recover: "/recover",
+  etoRecover: "/recover-eto",
```
Comment (huesforalice) [inline app/components/AppRouter.tsx]: Instinctively I would have "namespaced" this routes, so /eto/login and /investors/login. I think it looks nicer. Just a suggestion though.

File: app/components/AppRouter.tsx [hunk 5]
```diff
@@ -38,18 +45,40 @@ export const AppRouter: React.SFC = () => (
...
-    <OnlyAuthorizedRoute path={appRoutes.wallet} component={Wallet} />
-    <OnlyAuthorizedRoute path={appRoutes.kyc} component={Kyc} />
-    <OnlyAuthorizedRoute path={appRoutes.dashboard} component={Dashboard} exact />
-    <OnlyAuthorizedRoute path={appRoutes.settings} component={Settings} exact />
-    <OnlyAuthorizedRoute path={settingsRoutes.seedBackup} component={BackupSeed} exact />
-    <OnlyAuthorizedRoute path={appRoutes.eto} component={Eto} />
+    {/* only investors routes */}
+    <OnlyAuthorizedRoute path={appRoutes.wallet} investorComponent={Wallet} />
+    <OnlyAuthorizedRoute path={appRoutes.kyc} investorComponent={Kyc} />
+
```
Comment (huesforalice) [inline app/components/AppRouter.tsx]: This is quite nice.

**Pull request merged**